content/_guides/faq.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Frequently Asked Questions
+category: about-irc
+---
+
+- TOC
+{:toc}
+
+## You need to identify via SASL to use this server
+
+You have tried to connect from a
+[SASL access only range](guides/sasl#sasl-access-only-ip-ranges)
+
+## Cannot send to nick/channel
+
+## Cannot change nickname while banned on channel
+
+First, check that you are [registered](registration).
+
+If you already have an account then [identify](registration#logging-in) to it.
+
+Some channels require users to be registered to participate. This is because
+most botnets and disruptive people do not make accounts.
+
+If you are both registered and identified and still cannot join or talk, you
+might be [banned or quieted](faq#how-do-i-get-unbanned-or-unquieted)
+in that channel.
+
+## You must log in with services to message this user
+
+Some people do not wish to get messages from unregistered users. If you are
+not currently logged in then you will also be affected.
+
+This is probably **not** targetted at you personally.
+
+First, check if you are [registered](registration). If you do have an account
+then make sure you have [identified](registration#logging-in) to it.
+
+## How do I get unbanned or unquieted?
+
+If you have been banned from or quieted in a channel, you will need to resolve
+the issue with the people who run the channel. If you cannot remember who
+banned or quieted you, there are a few options.
+
+You can also try to find your ban/quiet, and the operator who set it, by
+viewingthe ban and quiet lists for the channel (this may output a lot of
+lines):
+
+```irc
+/mode #channelname bq
+```
+
+If that is too overwhelming, or if `PUBACL` (public access control lists) is
+off, or if all the bans are attributed to Libera.Chat domains, you can instead
+try to find out who runs the channel:
+
+```irc
+/msg chanserv info #channelname
+/msg chanserv flags #channelname
+```
+
+If the channel is set `PRIVATE` then you will not be allowed to do this.
+
+Your only remaining option is to find someone else who in the channel to who
+might help you get in contact.
+
+**Be considerate of other channels when doing this as it can be seen as
+dragging drama in from other places!**
+
+Remember, channel ops are **not** obligated to negotiate with you, and evading
+bans (sneaking back in) usually makes the matter worse.
+
+Asking network staff will not work as we **will not** override a ban or quiet
+set by a channel operator, and will not reveal information that is private. We
+also **will not** confiscate channels for "unfair" banning practices.
```

Comment:
Do we want to add a caveat for ToS violating activity?

Reply:
I think that would just tempt them to try make up tos violations.